stubs/oauthlib/oauthlib/oauth2/rfc6749/endpoints/pre_configured.pyi

```diff
@@ -1,75 +1,86 @@
-from _typeshed import Incomplete
+from collections.abc import Callable
 from typing import Any
 
+from oauthlib.common import Request
+
+from ..grant_types import (
+    AuthorizationCodeGrant,
+    ClientCredentialsGrant,
+    ImplicitGrant,
+    RefreshTokenGrant,
+    ResourceOwnerPasswordCredentialsGrant,
+)
+from ..request_validator import RequestValidator
+from ..tokens import BearerToken
 from .authorization import AuthorizationEndpoint
 from .introspect import IntrospectEndpoint
 from .resource import ResourceEndpoint
 from .revocation import RevocationEndpoint
 from .token import TokenEndpoint
 
 class Server(AuthorizationEndpoint, IntrospectEndpoint, TokenEndpoint, ResourceEndpoint, RevocationEndpoint):
-    auth_grant: Any
-    implicit_grant: Any
-    password_grant: Any
-    credentials_grant: Any
-    refresh_grant: Any
-    bearer: Any
+    auth_grant: AuthorizationCodeGrant
+    implicit_grant: ImplicitGrant
+    password_grant: ResourceOwnerPasswordCredentialsGrant
+    credentials_grant: ClientCredentialsGrant
+    refresh_grant: RefreshTokenGrant
+    bearer: BearerToken
     def __init__(
         self,
-        request_validator,
-        token_expires_in: Incomplete | None = None,
-        token_generator: Incomplete | None = None,
-        refresh_token_generator: Incomplete | None = None,
-        *args,
-        **kwargs,
+        request_validator: RequestValidator,
+        token_expires_in: int | Callable[[Request], int] | None = None,
+        token_generator: Callable[[Request], str] | None = None,
+        refresh_token_generator: Callable[[Request], str] | None = None,
+        *args: Any,  # actually, these are not used
+        **kwargs: Any,  # actually, these are not used
```

Comment:
If they are not used, we should use `(_typeshed.)Unused` instead of a comment. A potentially better option is to remove the arguments to prevent users from misspelling, e.g. `Server(token_genarator=...)` isn't an error with the current annotations.

Reply:
to be honest i already forgot why i preferred Any, but i think because subclasses can use them

but still I'll probably just remove them